### PR TITLE
Build SearchForJobs component and search functionality

### PR DESCRIPTION
**Pull request by wpiao**

### CODE REVIEW

File: components/SearchForJobs.tsx [hunk 1]
```diff
@@ -1,10 +1,69 @@
...
+import axios from "axios";
 
 export const SearchForJobs = () => {
+  const [jobs, onChangeJobs] = useState("");
+  const [location, onChangeLocation] = useState("");
+  const [data, setData] = useState([]);
+
+  const handleSearch = () => {
+    let formattedJobs = "";
+    for (const letter of jobs) {
```
Comment (raymond-sde) [inline components/SearchForJobs.tsx]: remove unnecessary formatting. url encoding takes care of the space for you

File: components/SearchForJobs.tsx [hunk 2]
```diff
@@ -1,10 +1,69 @@
...
+      if (letter === " ") {
+        formattedJobs += "+";
+      } else {
+        formattedJobs += letter;
+      }
+    }
+
+    let formattedLocation = "";
+    for (const letter of location) {
```
Comment (raymond-sde) [inline components/SearchForJobs.tsx]: same here

**Review by raymond-sde — APPROVED** [review]

File: components/SearchForJobs.tsx (hunk 1 above)
Comment (wpiao) [reply components/SearchForJobs.tsx]: Done

File: components/SearchForJobs.tsx (hunk 2 above)
Comment (wpiao) [reply components/SearchForJobs.tsx]: Done
